test_container.sh

```diff
@@ -0,0 +1,31 @@
+rm -rf TEMP
```

Comment:
It looks like this file is only here because `go` wasn't stubbed. Have a look at https://github.com/IBM-Blockchain-Starter-Kit/build-lib/blob/master/test/prepare-unstable.bats for an example of stubbing commands.

Reply:
The directory structure that the build-lib scripts  see when they are in the actual pipeline is highly different from what exists in build-lib. In fact the src/ directory here conflicts with the src/ directory of the scaffolding repository, so during prepare it is replaced with the scripts/ directory. Many of the build-lib scripts have paths referencing this scripts directory, and would fail without  it's existence. In order to simulate exactly the pipeline environment, I created this test_container script. Many of these tests aren't actually filled in with logic yet, mostly because we wanted to solve the directory madness first.

Reply:
I've created a new pull request ##26 which sets up the directory structure as it should exist in the build pipeline on a per-testcase basis, so anything left over from earlier tests shouldn't interfere with subsequent tests

I'll raise an issue for the src directory in the scaffold repository since it looks like that shouldn't be there anyway